docs/source/detailed/attributeList.rst

```diff
@@ -2,7 +2,7 @@ Alphabetical List of Attributes
 **************************************************
 
 A complete list of all attributes in the New Horizons Mission Dictionary, in alphabetical
-order, is available through the `PDS4 Data Dictionary <https://pds.nasa.gov/datastandards/documents/dd/current/PDS4_PDS_DD_1J00/webhelp/all/>`_
+order, is available through the `PDS4 Data Dictionary <https://pds.nasa.gov/datastandards/documents/dd/current/PDS4_PDS_DD_1L00/webhelp/all/>`_

```

Comment:
Should we keep the link pointing to 1.19 given that's the IM version this mission/dictionary uses?

Reply:
Not in this very particular case. This utility is manually maintained, so it is very unlikely than EN will retrofit it for us to add the new NH info to the 2yr old site. A user going here just wants the definitions, not the schema relationships, so the core schema version isn't relevant to the information a user will see for the mission namespace (because we build the same classes and attributes irrespective of IM version). Which reminds me, I should make sure that link exists...

Reply:
In fact, the site exists but of course the NH dictionary isn't in it. I'll make a note in the issues list to follow up on this with Jordan.